pkg/wwan/mmagent/mmdbus/client.go

```diff
@@ -1357,9 +1357,12 @@ func (c *Client) reconfigureEpsBearerIfNotRegistered(modemObj dbus.BusObject,
 	}
 	var currentSettings map[string]dbus.Variant
 	_ = getDBusProperty(c, modemObj, Modem3GPPPropertyInitialEpsBearer, &currentSettings)
+	maskedPasswd := interface{}("***")
+	maskedVariantPasswd := dbus.MakeVariant(maskedPasswd)
 	c.log.Warnf("Modem %s is failing to register, "+
 		"trying to apply settings %+v for the initial EPS bearer (previously: %+v)",
-		modemObj.Path(), newSettings, currentSettings)
+		modemObj.Path(), maskPassword(newSettings, maskedPasswd),
```

Comment:
@milan-zededa , 100% agree about the needs of masking the password. At other hand, we will reduce the information for debugging. I'm not suggesting any changes here... but perhaps in the future we could use something like this go package: https://pkg.go.dev/github.com/ggwhite/go-masker#section-readme that has different options for masking passwords, so we could use some hash (based on the device certificate, for instance) that would allow us the recovery of the password without print it in plain text. Btw, the one that you are implementing is also available but it returns more asterisks: `************` instead of `***`, what do you think?

Reply:
That package allows to mark struct fields for masking. However, here we have a map. Therefore I had to write my own method.
But note that the encrypted version of the password is available in `/run/nim/WwanConfig.json`, which is collected by collect-info.sh. Using ecdh certificate it can be decrypted if needed for debugging purposes.

Reply:
ok, got it, thanks for the explanation....